django/urls/resolvers.py

```diff
@@ -371,6 +371,9 @@ def lookup_str(self):
         callback = self.callback
         if isinstance(callback, functools.partial):
             callback = callback.func
+        if hasattr(callback, "view_class"):
+            klass = callback.view_class
+            return f"{klass.__module__}.{klass.__qualname__}"
```

Comment:
This is an undocumented URL resolver method that seems to only exist to support the admindocs functionality.

Reply:
Do you think it's worth making a ticket to see if we could move this functionality into `admindocs`? I've never seen a project using `admindocs` and it seems wasteful to build the whole reverse mapping data structure for projects not using it.

Reply:
I guess there is a fair amount of wasted effort for the majority(?) of projects that aren't using admindocs, so perhaps we could have a ticket for it to investigate the possibility of moving the admindocs specific-stuff out.

Reply:
https://code.djangoproject.com/ticket/32565#ticket